css/fieldmanager-group-tabs.css

```diff
@@ -171,3 +188,29 @@
 	-webkit-box-shadow: none;
 	box-shadow: none;
 }
+
+@media screen and (min-width: 575px) {
+
+	.fm-tabbed-vertical .fm-tab-bar {
+		width: 175px;
+		float: left;
+		margin: 0 0 0 -12px;
```

Comment:
I've checked this out locally and the `-12px` value causes vertical tabs to be offset incorrectly:

![image 2018-05-03 at 11 58 29](https://user-images.githubusercontent.com/1757914/39588347-a1c75f58-4ec9-11e8-9b13-bc214446ed8a.png)

Changing this to `margin: 0 0 0 1px;` fixes the issue for me. The `1px` is to account for the border of a field group:

![image 2018-05-03 at 12 01 37](https://user-images.githubusercontent.com/1757914/39588493-dafdac96-4ec9-11e8-823e-44b31019f8d8.png)